views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
this is better done as a new step, preserving the original matrix:
```
And, by using sign manipulation, we can rewrite this matrix into a form that has Bernstein terms on each row, using "z" rather than the "t":

[ 1 0 0 ; -(1-z) z 0 ; ...] = [your Bernstein matrix]
```

But we do need to point out here that this is *only* possible if we don't care about the directionality of `z`, since the maths may be symmetric, but turning `(1-z)²` into `(z-1)²` changes the geometric meaning: instead of splitting left to right, we'll now be splitting right to left. That's pretty important to know.